assets/scss/navbar.scss

```diff
@@ -0,0 +1,249 @@
+.navbar {
+  background-color: $white;
+  box-shadow: 0 2px 69px 0 rgba(13, 13, 30, 0.04);
+  font-size: 18px;
+  font-weight: 600;
+  min-height: 75px;
+  width: 100%;
+
+  @include m-up {
+    display: flex;
+    height: 75px;
+    padding: 0 40px;
+  }
+
+  &__header {
+    align-items: center;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    min-height: 75px;
+    padding: 0 20px;
+
+    @include m-up {
+      margin-right: 80px;
+      min-width: 110px;
+      padding: 0;
+    }
+  }
+
+  &__brand {
+    color: $gray;
+  }
+
+  &__menu {
+    align-items: center;
+    display: none;
+    flex-direction: column;
+    padding: 20px 0 40px;
+
+    &--visible {
+      display: flex;
+    }
+
+    @include m-up {
+      align-items: center;
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      padding: 0;
+      width: 100%;
+    }
+  }
+
+  &__menu-items {
+    text-align: center;
+
+    li {
+      margin: 40px 0 0;
+
+      &:first-of-type {
+        margin: 0;
+      }
+
+      &:hover,
+      &:active {
+        a {
+          color: $accent;
+        }
+      }
+    }
+
+    @include m-up {
+      display: flex;
+
+      li,
+      a {
+        margin: 0 !important; // :(
+      }
+
+      li {
+        margin-right: 40px !important;
+      }
+
+      li:last-of-type {
+        margin: 0 !important;
+      }
+    }
+
+    @include l-up {
```

Comment:
Each include of mediaquery should be placed directly in each selector, avoid using selectors inside inculding mediaqueries